inference-engine/tests/functional/plugin/cpu/single_layer_tests/convolution.cpp

```diff
@@ -770,4 +717,91 @@ INSTANTIATE_TEST_SUITE_P(smoke_Conv_winograd, ConvolutionLayerCPUTest,
 
 } // namespace winograd
 
+/* ============= U8/I8 Convolution ============= */
+namespace int8 {
+
+const std::vector<fusingSpecificParams> fusingParamsSetI8{
+        emptyFusingSpec,
+//        // activations
+        fusingElu,
+        fusingSigmoid,
+        fusingPReluPerChannel,
+        fusingSwish,
+        fusingMish,
+//        // other patterns
+        fusingSumEluFQ,
+        fusingSum,
+        fusingAddPerChannel
+};
+
+const std::vector<CPUSpecificParams> CPUParams_2D_I8 = {
+        conv_gemm_2D_nspc,
+        conv_sse42_2D_nspc,
+        conv_avx2_2D_nspc,
+        conv_avx512_2D_nspc
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_Conv_2D_I8, ConvolutionLayerCPUTest,
+        ::testing::Combine(
+                ::testing::Combine(
+                        convParams_ExplicitPadding_GEMM_2D,
+                        ::testing::Values(Precision::FP32),
+                        ::testing::Values(Precision::U8 /*, Precision::I8*/), // i8 primitives are disabled in oneDNN fork
+                        ::testing::Values(Precision::FP32, Precision::U8, Precision::I8),
+                        ::testing::Values(Layout::ANY),
+                        ::testing::Values(Layout::ANY),
+                        ::testing::Values(std::vector<size_t >({ 2, 12, 7, 7 })),
+                        ::testing::Values(CommonTestUtils::DEVICE_CPU)),
+                ::testing::ValuesIn(filterCPUInfoForDevice(CPUParams_2D_I8)),
+                ::testing::ValuesIn(fusingParamsSetI8),
+                ::testing::Values(cpuEmptyPluginConfig)),
+         ConvolutionLayerCPUTest::getTestCaseName);
+
+const std::vector<CPUSpecificParams> CPUParams_3D_I8 = {
+        conv_gemm_3D_nspc,
+        conv_sse42_3D_nspc,
+        conv_avx2_3D_nspc,
+        conv_avx512_3D_nspc
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_Conv_3D_I8, ConvolutionLayerCPUTest,
+        ::testing::Combine(
+                ::testing::Combine(
+                        convParams_ExplicitPadding_GEMM_3D,
+                        ::testing::Values(Precision::FP32),
+                        ::testing::Values(Precision::U8 /*, Precision::I8*/), // i8 primitives are disabled in oneDNN fork
+                        ::testing::Values(Precision::FP32, Precision::U8, Precision::I8),
+                        ::testing::Values(Layout::ANY),
+                        ::testing::Values(Layout::ANY),
+                        ::testing::Values(std::vector<size_t >({ 2, 12, 7, 7, 7 })),
```

Comment:
12 channels does not form a full vector on avx512 VNNI, m.b. it makes sense to have a shape that contains at least one full SIMD vector and some fractional part to cover more possible code paths? Here in in the other new test instances.